### PR TITLE
Convert App.js => App.tsx

### PR DESCRIPTION
**Pull request by walmat**
Going to be undertaking this project in my downtime. Starting with the top-level tree hierarchy and going down.

### CODE REVIEW

File: src/navigation/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Platform } from 'react-native';
+
+export default Platform.select({
+  ios: require('./Routes.ios'),
+  android: require('./Routes.ios'),
```
Comment (jinchung) [inline src/navigation/Routes.tsx:5]: did you mean `Routes.android` ?

Comment (walmat) [reply src/navigation/Routes.tsx:5]: 😅

File: src/navigation/initialRoute.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { createContext } from 'react';
+import Routes from './routesNames';
+
+export const InitialRouteContext = createContext<string>(Routes.WELCOME_SCREEN);
```
Comment (jinchung) [inline src/navigation/initialRoute.ts:4]: curious about the change from `null` as the default value to the `WELCOME_SCREEN` here if it's needed or if it'll work the same with `null` as it was and just passed in Context type `<string | null>`

Comment (walmat) [reply src/navigation/initialRoute.ts:4]: Didn't see any change in behavior personally

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**